src/btree.c

```diff
@@ -3295,7 +3323,20 @@ btree_print_tree(platform_log_handle *log_handle,
                  btree_config        *cfg,
                  uint64               root_addr)
 {
-   btree_print_subtree(log_handle, cc, cfg, root_addr);
+   btree_print_subtree(log_handle, cc, cfg, root_addr, PAGE_TYPE_BRANCH);
+}
+
+/*
+ * Routine to print a BTree of page-type 'type' starting from root_addr
+ */
+static void
+btree_print_tree_as(platform_log_handle *log_handle,
```

Comment:
Routine is very similar to `btree_print_tree()`, but just takes an extra `page_type` arg.

I kept the name short with `_as` suffix ... 

I thought about `btree_print_tree_of_type` or `_with_type` but neither was appealing beyond `_as()` variant, without adding too long a function name.

Reply:
I would delete the _as suffix.

Reply:
(robj) - Avoid the `_as()` suffix. See if the top-level `btree_print_tree()` itself could take a page-type and just use that.

Just have `btree_print_tree_as()` replace `btree_print_tree()` and the latter takes a page-type argument.

Reply:
Made this change as discussed / suggested.